data/field.go

```diff
@@ -196,6 +196,13 @@ func (f *Field) Set(idx int, val interface{}) {
 	f.vector.Set(idx, val)
 }
 
+// SetConcreateAt sets the Fields's value to index idx to the val
+// if the Field is not nullable or to the pointer to val if it is nullable
+// It will panic if the undelying type of val does not match the element concrete type of the Field
+func (f *Field) SetConcreateAt(idx int, val interface{}) {
+	f.vector.Set(idx, val)
```

Comment:
Should be f.vector.SetConcreteAt I think

Reply:
good point!
I've used `vector. SetConcreateAt` instead so I didn't noticed it.